src/profession.cpp

```diff
@@ -221,7 +221,7 @@ void profession::check_item_definitions( const itypedecvec &items ) const
 {
     for( auto &itd : items ) {
         if( !item::type_is_defined( itd.type_id ) ) {
-            debugmsg( "profession %s: item %s does not exist", id.c_str() , itd.type_id.c_str() );
+            debugmsg( "profession %s: item %s does not exist", id.c_str(), itd.type_id.c_str() );
```

Comment:
I have no idea.

By the time I submitted this, PRs were no longer failing en-masse.

The situation here is very similar to #20093.